sysutils/pfSense-pkg-nut/files/usr/local/pkg/nut.xml

```diff
@@ -28,7 +28,7 @@
 	]]>
 	</copyright>
 	<name>nut</name>
-	<version>2.8.0_4</version>
+	<version>2.8.0_5</version>
```

Comment:
This version didn't follow the other change. Though honestly the tag doesn't need to be in this file any longer, the version is handled entirely in the Makefile variables now.

There is still a version tag in `files/usr/local/share/<pkg name>/info.xml` but it should be using a macro like `<version>%%PKGVERSION%%</version>` so you don't ever have to update it by hand.

Reply:
My bad. Thanks for the catch. I'll change it to %%PKGVERSION%%.

Reply:
Done